frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -446,12 +450,33 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
                 </Card>
               </Radio.Button>
             </Tooltip>
+            <Tooltip title="Coming soon">
+              <Radio.Button
+                className="aIJobSelection"
+                checked={aIJobModalState === "align_sections"}
+                disabled={!Store.getState().activeUser?.isSuperUser}
+                onClick={() => Store.dispatch(setAIJobModalStateAction("align_sections"))}
+              >
+                <Card bordered={false}>
+                  <Space direction="vertical" size="small">
+                    <Row className="ai-job-title">Align Sections</Row>
+                    <Row>
+                      <img
+                        src={`/assets/images/${jobNameToImagePath.align_sections}`}
+                        alt={"Mitochondria detection example"}
```

Comment:
This should be renamed to something like
```suggestion
                        alt={"Example of improved alignment of slices"}
```